### ISSUE TITLE
@turf/nearest-point throws "TypeError: helpers_1.degreesToRadians is not a function"

### ISSUE THREAD

**Issue by barbalex** [Turfjs/turf#1419]
I have been using `@turf/nearest` in v4.7.3.
Now I wanted to update to `@turf/nearest-point` v6.0.1.

As far as I can tell both methods use the exact same api.
So I use the new instead of the (flawlessy working) old and constantly get this error:

```
index.js:33 Uncaught (in promise) TypeError: helpers_1.degreesToRadians is not a function
    at Object.distance [as default] (index.js:33)
    at index.js:43
    at Object.featureEach (index.js:322)
    at nearestPoint (index.js:42)
    at webpackJsonp../src/modules/getNearestTpop.js.__webpack_exports__.a (getNearestTpop.js:83)
```

This is my module: https://github.com/barbalex/apf2/blob/3155c437d9b7ea2ece2ceafab06f9a42efd5ccca/src/modules/getNearestTpop.js

Am I doing something wrong or is this a bug?

**Comment by rowanwins** [comment]
Hi @barbalex 

There have been a few issues with publishing in the v6 modules. Probably best to roll back to something in the v5 series for the timebeing. 

Cheers

**Comment by barbalex** [comment]
@rowanwins thanks, will do

**Comment by ahaw021** [comment]
i have updated a few on my local machine but it would be good to get sorted :D 

original 

![image](https://user-images.githubusercontent.com/26753809/44448828-c6930e00-a62f-11e8-9003-3afa28a4447f.png)

updated version

![image](https://user-images.githubusercontent.com/26753809/44448857-da3e7480-a62f-11e8-8a25-72ca94673333.png)

**Comment by mfedderly** [comment]
This should be fixed by the 6.2.0 and later releases, if it isn't please reopen the ticket.
